### PR TITLE
Add support for Redstone 5

### LINKED ISSUE #805: Add support for Redstone 5

### PR DESCRIPTION
**Pull request by tfenster**
Add support for Redstone 5 which should generate Dockerfiles referencing -1809 images

Fixes #805

### CODE REVIEW

**Comment by tfenster** [comment]
Didn't see the tests, will implement that later

**Comment by StephenWeatherford** [comment]
cc @philliphoff

File: configureWorkspace/configure_dotnetcore.ts [hunk 1]
```diff
@@ -36,8 +36,10 @@ const DotNetCoreSdkImageFormat = "microsoft/dotnet:{0}.{1}-sdk{2}";
 
 function GetWindowsImageTag(): string {
     // The host OS version needs to match the version of .NET core images being created
-    if (!isWindows() || isWindows10RS4OrNewer()) {
+    if (!isWindows() || isWindows10RS5OrNewer()) {
         // If we're not on Windows (and therefore can't detect the version), assume a Windows RS4 host
```
Comment (StephenWeatherford) [inline configureWorkspace/configure_dotnetcore.ts]: update comment

**Comment by StephenWeatherford** [comment]
Thanks, will await the tests.  We have a release of the extension going out in a about a week.

File: configureWorkspace/configure_dotnetcore.ts (hunk 1 above)
Comment (tfenster) [reply configureWorkspace/configure_dotnetcore.ts]: done, thanks

**Comment by tfenster** [comment]
@StephenWeatherford Test now seems fine. Did I miss anything?

**Comment by StephenWeatherford** [comment]
In Ulm, um Ulm, und um Ulm herum…  :-)

**Pull request merged**